### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by automata**

### CODE REVIEW

**Comment by bergie** [comment]
At least couple of things to be done:

* Make compatible with Chai 3.x (`to.be.an('object')`) requires disambiguation
* Load components in tests via NoFlo componentloader

**Comment by automata** [comment]
@bergie can you review, pls?

File: package.json [hunk 1]
```diff
@@ -17,39 +17,44 @@
...
+    "noflo-filesystem": "~1.1.1",
     "noflo-groups": "~0.1.10",
     "noflo-objects": "~0.2.0",
     "noflo-strings": "~0.2.0",
     "read-chunk": "^2.0.0",
-    "request": "^2.69.0",
+    "request": "^2.79.0",
     "rgbquant": "1.1.2",
-    "superagent": "2.2.0",
+    "superagent": "3.3.2",
```
Comment (bergie) [inline package.json]: Might be best to add wildcard?

File: package.json [hunk 2]
```diff
@@ -17,39 +17,44 @@
...
-    "mocha": "^2.4.5"
+    "grunt-contrib-watch": "^1.0.0",
+    "grunt-mocha-phantomjs": "^4.0.0",
+    "grunt-mocha-test": "^0.13.2",
+    "grunt-noflo-browser": "^1.1.4",
+    "grunt-noflo-manifest": "^0.1.12",
+    "json-loader": "^0.5.4",
+    "mocha": "^3.2.0"
   },
   "keywords": [],
```
Comment (bergie) [inline package.json]: Add `noflo` keyword

**Comment by automata** [comment]
@bergie done, merging.

**Pull request merged**